.github/workflows/android.yml

```
@@ -31,17 +30,10 @@ jobs:
         GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES }}
       run: echo "$GOOGLE_SERVICES_JSON" > ./app/google-services.json
 
-    - name: Cache Gradle
-      uses: actions/cache@v3
+    - name: Setup Gradle and Build
+      uses: gradle/gradle-build-action@v2
       with:
-        path: |
-          ~/.gradle/caches
-          ~/.gradle/wrapper
-        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
-        restore-keys: |
-          ${{ runner.os }}-gradle-
-        
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Build with Gradle
-      run: ./gradlew build
+        arguments: build
+        # Only write to the cache for builds on the 'main' and 'release' branches. (Default is 'main' only.)
```

```suggestion
        # Only write to the cache for builds on the 'main' and 'develop' branches. (Default is 'main' only.)
```
@SangsooNam 제가 현재 release branches를 운용하지 않고 main과 develop으로 작업하기에 멘션이 바뀌어야 될 것 같습니다!